app/core/src/test/java/com/fsck/k9/helper/ListUnsubscribeHelperTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.fsck.k9.helper
+
+import com.fsck.k9.K9RobolectricTest
+import com.fsck.k9.mail.internet.MimeMessage
+import org.junit.Assert
+import org.junit.Test
+
+class ListUnsubscribeHelperTest : K9RobolectricTest() {
```

Comment:
This test only needs Robolectric, but none of the K-9 machinery running. So these tests will run a bit faster if we inherit from `RobolectricTest`.